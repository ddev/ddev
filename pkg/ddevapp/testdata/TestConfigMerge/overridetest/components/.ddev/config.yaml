
# Use some non-default settings to make sure they get through
database:
  type: mariadb
  version: 10.5
php_version: 5.6
webserver_type: apache-fpm
nfs_mount_enabled: false
