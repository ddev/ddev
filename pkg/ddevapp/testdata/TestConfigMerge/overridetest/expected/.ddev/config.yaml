
database:
  type: mariadb
  version: 10.5
php_version: 5.6
webserver_type: apache-fpm
webimage_extra_packages: [php7.4-tidy]
nfs_mount_enabled: true
